diagnostic.md

```diff
@@ -93,5 +93,9 @@ var teamMember = {
 
 What is the value of `teamMember['hobby']`?  Of `teamMember['name']`?  Of
 `teamMember['name']['surname']`?
+'exercise'
 
-_Replace this text with your answer._
+given:
+surname:
+
+'Weeks'
```

Comment:
You got the first and third ones correct.  teamMember['name'] is actually a nested object, { given: 'Jason', surname: 'Weeks' }.  You gave given and surname, but those are just the keys of the nested object.